packages/cozy-harvest/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cozy-harvest",
+  "version": "0.1.0",
+  "description": "Provides logic, modules and components for Cozy's harvest applications.",
+  "main": "dist/index.js",
+  "author": "Cozy",
+  "license": "MIT",
+  "homepage": "https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest/README.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cozy/cozy-libs.git"
+  },
+  "bugs": {
+    "url": "https://github.com/cozy/cozy-libs/issues"
+  },
+  "scripts": {
+    "test": "jest --verbose --coverage",
+    "build": "babel src -d dist",
+    "prepublishOnly": "yarn build",
+    "watch": "yarn build --watch"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.1.2",
+    "babel-preset-cozy-app": "^1.2.1",
+    "react": "16.6.1",
+    "react-final-form": "3.6.7",
```

Comment:
I could be wrong but it seems that `react-final-form` is not needed in this initial version.